asllib/doc/ASLTypingReference.tex

```diff
@@ -1579,6 +2440,9 @@ \section{TypingRule.ECall \label{sec:TypingRule.ECall}}
     parameters \texttt{eqs};
   \item \texttt{name', args', eqs', ty} is the result of annotating the call of
     that subprogram in \texttt{env};
+\begin{comment}
```

Comment:
as discussed please edit as needed to have matching names in the prose, maths and code everywhere

Reply:
I can match the names in the prose and math, but full consistency with the names in the code is not possible right now, since the code uses shadowing so variable names are re-used. For that, we need to change the code itself.

Reply:
yes, that is what I am asking you to do please.